builder/tencentcloud/cvm/image_config.go

```diff
@@ -31,8 +31,7 @@ type TencentCloudImageConfig struct {
 	// accounts that will be shared to
 	// after your image created.
 	ImageShareAccounts []string `mapstructure:"image_share_accounts" required:"false"`
-	// Do not check region and zone when validate.
-	SkipValidation bool `mapstructure:"skip_region_validation" required:"false"`
+	skipValidation     bool     `mapstructure:"_"`
```

Comment:
Mapstructure tag is not needed. 
```suggestion
	skipValidation     bool   
```